test/groovy/ToolValidateTest.groovy

```diff
@@ -269,8 +248,8 @@ class ToolValidateTest extends BasePipelineTest {
     }
 
 
-    private createPipeline(pipelinePath){
-        new File(pipelinePath) <<   """
+    private createPipeline(){
```

Comment:
I did call it createPipeline because it was creating a File, since it is not creating the file no more inside this method, could you please change it to defaultPipeline, if you have to apply some changes?

Reply:
done